R/conversion.R

```diff
@@ -154,7 +154,7 @@ py_to_r.datetime.datetime <- function(x) {
 #' @export
 r_to_py.Date <- function(x, convert = FALSE) {
 
-  datetime <- import("datetime", convert = convert)
+  datetime <- import("datetime", convert = FALSE)
```

Comment:
Yes, that's correct. If you always want the conversion to yield an R object then you should explicitly pass `convert = FALSE`